src/app/shell/personal-cabinet/parent/create-application/create-application.component.ts

```diff
@@ -42,11 +44,26 @@ export class CreateApplicationComponent implements OnInit {
     private route: ActivatedRoute,
     private matDialog: MatDialog) { }
 
+  /**
+    * This method create new Navigation button
+    */
+  creatNavPath(name:string, isActive:boolean, disable:boolean): Nav[] {
+    return [
+      {name:'Головна', path:'/', isActive:true, disable:false},
```

Comment:
if main page is always presented in the path we may move it to the navigation component and attach to dispatched items